tests/test_redis_aggregator.py

```diff
@@ -74,5 +75,11 @@ def on_batch_response(self, market_stats):
 selected = pv.select_aggregator(aggregator.aggregator_uuid)
 logging.info(f"SELECTED: {selected}")
 
+
+redis_market = RedisMarketClient('house-2')
+market_slot_string_2 = today().add(minutes=60).format(DATE_TIME_FORMAT)
```

Comment:
why `_2` ?

Reply:
Just quick variable naming and copy/pasting. Will update with better name.